### PR TITLE
Update orka from 1.2.0 to 1.3.0

### PR DESCRIPTION
**Pull request by Fatme**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/orka.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'orka' do
-  version '1.2.0,6924a4c7'
-  sha256 '3be344b879b01b54524065b1e2026633b77260d79823f23860dcc8149f32d127'
+  version '1.3.0,c710094'
+  sha256 '4178c1f16d82e9b93a825419c6228b7d8237b609082537732acef387ee5914f5'
 
   # cli-builds-public.s3-eu-west-1.amazonaws.com/official/ was verified as official when first introduced to the cask
-  url "https://cli-builds-public.s3-eu-west-1.amazonaws.com/official/#{version.before_comma}/#{version.after_comma}/macos/orka.zip"
+  url 'https://cli-builds-public.s3-eu-west-1.amazonaws.com/official/1.3.0/c710094/macos/orka.zip'
```
Comment (miccal) [inline Casks/orka.rb:6]: Please revert this change  -- interpolated `url`'s that use `version` are easier to maintain.

Comment (Fatme) [reply Casks/orka.rb:6]: This was changed by `brew cask style --fix {{cask_file}}` command. 
It seems it is styling rule.
